integration/fuse/src/main/java/alluxio/fuse/AlluxioJniFuseFileSystem.java

```diff
@@ -336,7 +336,10 @@ private int readdirInternal(String path, long buff, long filter, long offset,
   @Override
   public int open(String path, FuseFileInfo fi) {
     final int flags = fi.flags.get();
-    boolean overwrite = OpenFlags.valueOf(flags) == OpenFlags.O_WRONLY;
+    // TODO(maobaolong): Add an option to decide whether reject rw flag
+    //  or fallback to truncate
+    boolean overwrite = OpenFlags.valueOf(flags) == OpenFlags.O_WRONLY
+        || (flags & 0b11) != 0;
```

Comment:
can you add a description for what 0b11 means?

Reply:
(flags & 0b11) != 0; why doing this operation?
what flags included etc

Reply:
Wondering if the flags falls into one enum, comparing the enum, that could be more clear.